### DISCUSSION TITLE
Pastas on Conda Forge

### DISCUSSION THREAD

**Discussion by raoulcollenteur** [pastas/pastas#279] [General]
Just found out that Pastas can also be installed from Conda Forge: https://anaconda.org/conda-forge/pastas

No idea how, but this seems to be triggered automatically, nice :)

**Comment by Hugovdberg** [comment]
https://github.com/conda-forge/pastas-feedstock is the feedstock that updates the `pastas` package on `conda-forge`, it looks like it was created by @jentjr.

**Reply by raoulcollenteur** [reply]
Nice @Hugovdberg, and thanks @jentjr for adding the feedstock!
